app/src/main/res/layout/fragment_social_settings.xml

```diff
@@ -69,35 +69,18 @@
                     android:id="@+id/btnAddFriend"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginBottom="@dimen/margin_padding_size_38"
+                    android:layout_marginTop="@dimen/margin_padding_size_26"
+                    android:layout_marginEnd="144dp"
```

Comment:
No. devices have different sizes. This solution is not always (only on devices that are exactly as wide as your phone) centered. Use Constraint Layout to fix this issue.

Tipp: There is a icon on the bottom right of the live render while designing:
<img width="100" alt="image" src="https://github.com/StianJohnsen/dte2503_team_3_repo/assets/57566364/9c99f24d-3afa-472c-b4ce-4a58de12111b">
Drag this around to try different screen sizes